app/ui/browser/actions/external.js

```diff
@@ -11,9 +11,9 @@ specific language governing permissions and limitations under the License.
 */
 
 import {
-  createTab, setUserTypedLocation, closeTab, navigatePageTo,
+  createTab, setUserTypedLocation, closeTab,
```

Comment:
Prefer `import *`.  Less to update, easier to grep for, and better WebStorm linking.